spring-kafka/src/main/java/org/springframework/kafka/core/KafkaAdmin.java

```diff
@@ -389,7 +390,7 @@ public Map<String, TopicDescription> describeTopics(String... topicNames) {
 		}
 	}
 
-	AdminClient createAdmin() {
+	protected Admin createAdmin() {
```

Comment:
This will now be part of the API - we need to add a `@since` tag.

Reply:
Added a JavaDoc, thanks.